src/SDKs/CognitiveServices/dataPlane/Search/Microsoft.CognitiveServices.Search/Microsoft.CognitiveServices.Search.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.CognitiveServices.Search</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Search APIs.</Description>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <DefineConstants>$(DefineConstants);CODESIGN</DefineConstants>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyName>Microsoft.CognitiveServices.Search</AssemblyName>
+    <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;EntitySearch API;REST HTTP client;EntitySearch SDK;netcore451511</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the Cognitive Services Search SDK. Included with this release is support for EntitySearch API.</PackageReleaseNotes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="Customizations\" />
```

Comment:
@stefanvalianu you do not need this as part of your .csproj, your new directory structure/files will still be picked up.
We are trying to keep the project file as clean as possible.